app/services/authorization/enterprise_service.rb

```diff
@@ -37,7 +37,7 @@ def initialize(token)
   # Return a true ServiceResult if the token contains this particular action.
   def call(action)
     allowed =
-      if token.nil? || token.token_object.nil? || token.expired?
+      if OpenProject::Configuration.ee_manager_visible? && (token.nil? || token.token_object.nil? || token.expired?)
```

Comment:
@oliverguenther OK, what do you propose? The purpose is clear, right? I want to be able to ask one thing and know if I am allowed to use a feature or not.